schema/partner_artist.js

```diff
@@ -61,3 +69,12 @@ const PartnerArtist = {
 }
 
 export default PartnerArtist
+
+// The below can be used as the connection from an artist to its partners.
+// The edge is the PartnerArtist relationship, with the node being the partner.
+export const PartnerArtistConnection = connectionDefinitions({
```

Comment:
This connection definition was the key part. You can customize the edge/node type and fields for a connection...but then you may need to do some mucking when you're munging your API response into the form the connection expects.